RecoTauTag/RecoTau/plugins/PATTauIDEmbedder.cc

```diff
@@ -0,0 +1,95 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/PatCandidates/interface/PATTauDiscriminator.h"
+#include "DataFormats/PatCandidates/interface/Tau.h"
+#include "FWCore/Utilities/interface/transform.h"
+
+class PATTauIDEmbedder : public edm::stream::EDProducer<>
```

Comment:
I'm happy to do this move, but will do it not before this evening. I need a new setup to be able to run my standard tests upon changes for this.